config/ucx.conf

```diff
@@ -12,3 +13,15 @@ UCX_IB_SEND_OVERHEAD=bcopy:5ns,cqe:50ns,db:400ns,wqe_fetch:350ns,wqe_post:100ns
 UCX_MM_SEND_OVERHEAD=am_short:40ns,am_bcopy:220ns
 UCX_MM_RECV_OVERHEAD=am_short:40ns,am_bcopy:220ns
 UCX_RCACHE_OVERHEAD=360ns
+
+[AMD Rome]
+CPU model=Rome
+UCX_DISTANCE_BW=phb:auto,node:17000MBs,sys:5100MBs
```

Comment:
could it be something like
`UCX_DISTANCE_BW=auto,node:17000MBs,sys:5100MBs`
so what is not specified explicitly is auto